### PR TITLE
fix: Grouped Detailslist's collapseAllVisibility=hidden needs correct spacers and role=cell nodes

### PR DESCRIPTION
**Pull request by smhigley**
Fixes [15721](https://dev.azure.com/microsoftdesign/fluent-ui/_workitems/edit/15721)

## Previous Behavior

Previously, we fully removed the header cell with the collapse all button when the `collapseAllVisibility` was set to hidden, which resulted in an incorrect number of column headers for the content, as well as misaligned content cells, both visually and semantically.

screenshot of grouped detailsList showing group headers misaligned with the column headers

## New Behavior

Now when `collapseAllVisibility` is hidden, the header's collapse all button is gone, but an empty cell is present to ensure the correct number of column headers, and to have the column headers line up with their columns.

Cells in each row are also given the correct spacing and empty cells.

screenshot of grouped detailsList showing no header expand button, but correct spacing

### CODE REVIEW

File: packages/react/src/components/DetailsList/DetailsRow.base.tsx [hunk 1]
```diff
@@ -363,7 +363,7 @@ export class DetailsRowBase extends React.Component<IDetailsRowBaseProps, IDetai
         <GroupSpacer
           indentWidth={indentWidth}
           role="gridcell"
-          count={groupNestingDepth! - (this.props.collapseAllVisibility === CollapseAllVisibility.hidden ? 1 : 0)}
+          count={groupNestingDepth! === 0 ? -1 : groupNestingDepth!}
```
Comment (smhigley) [inline packages/react/src/components/DetailsList/DetailsRow.base.tsx:366]: This is the simplified result of swapping out `this.props.collapseAllVisibility === CollapseAllVisibility.hidden` for `groupNestingDepth! > 0`, similar to the change in DetailsHeader.

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/2ae8cc79c87d510b6ad5afd01b77c6a89f5debd9'>2ae8cc79c87d510b6ad5afd01b77c6a89f5debd9</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/BaseButton-mount_base.html">1417</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/BaseButton-mount.html">1378</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/Breadcrumb-mount_base.html">3451</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/Breadcrumb-mount.html">3469</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/Checkbox-mount_base.html">3044</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/Checkbox-mount.html">3081</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/CheckboxBase-mount_base.html">2783</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/CheckboxBase-mount.html">2738</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/ChoiceGroup-mount_base.html">5126</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/ChoiceGroup-mount.html">5194</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/ComboBox-mount_base.html">1474</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/ComboBox-mount.html">1493</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/CommandBar-mount_base.html">11143</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/CommandBar-mount.html">11174</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/ContextualMenu-mount_base.html">12592</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/ContextualMenu-mount.html">12560</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/DefaultButton-mount_base.html">1620</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/DefaultButton-mount.html">1636</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/DetailsRow-mount_base.html">4213</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/DetailsRow-mount.html">4150</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/DetailsRowFast-mount_base.html">4382</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/DetailsRowFast-mount.html">4206</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/DetailsRowNoStyles-mount_base.html">3991</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/DetailsRowNoStyles-mount.html">4017</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/Dialog-mount_base.html">3661</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/Dialog-mount.html">3654</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/DocumentCardTitle-mount_base.html">660</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/DocumentCardTitle-mount.html">681</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/Dropdown-mount_base.html">3770</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/Dropdown-mount.html">3768</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/FocusTrapZone-mount_base.html">2375</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/FocusTrapZone-mount.html">2382</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/FocusZone-mount_base.html">2300</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/FocusZone-mount.html">2307</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/GroupedList-mount_base.html">2174</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/GroupedList-mount.html">2477</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/GroupedList-virtual-rerender_base.html">1349</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/GroupedList-virtual-rerender.html">1328</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">1940</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/GroupedList-virtual-rerender-with-unmount.html">1941</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/GroupedListV2-mount_base.html">669</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/GroupedListV2-mount.html">646</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/GroupedListV2-virtual-rerender_base.html">641</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/GroupedListV2-virtual-rerender.html">639</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">684</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">678</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/IconButton-mount_base.html">2247</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/IconButton-mount.html">2262</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/Label-mount_base.html">856</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/Label-mount.html">850</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/Layer-mount_base.html">5084</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/Layer-mount.html">5123</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/Link-mount_base.html">946</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/Link-mount.html">967</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/MenuButton-mount_base.html">1931</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/MenuButton-mount.html">1944</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/MessageBar-mount_base.html">2669</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/MessageBar-mount.html">2715</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/Nav-mount_base.html">3835</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/Nav-mount.html">3811</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/OverflowSet-mount_base.html">1613</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/OverflowSet-mount.html">1585</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/Panel-mount_base.html">2947</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/Panel-mount.html">2979</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/Persona-mount_base.html">1528</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/Persona-mount.html">1561</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/Pivot-mount_base.html">1909</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/Pivot-mount.html">1938</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/PrimaryButton-mount_base.html">1783</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/PrimaryButton-mount.html">1777</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/Rating-mount_base.html">8150</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/Rating-mount.html">8159</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/SearchBox-mount_base.html">1762</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/SearchBox-mount.html">1743</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/Shimmer-mount_base.html">3344</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/Shimmer-mount.html">3263</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/Slider-mount_base.html">2447</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/Slider-mount.html">2448</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/SpinButton-mount_base.html">5438</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/SpinButton-mount.html">5452</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/Spinner-mount_base.html">942</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/Spinner-mount.html">964</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/SplitButton-mount_base.html">3654</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/SplitButton-mount.html">3601</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/Stack-mount_base.html">1000</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/Stack-mount.html">972</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/StackWithIntrinsicChildren-mount_base.html">2853</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/StackWithIntrinsicChildren-mount.html">2872</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/StackWithTextChildren-mount_base.html">5762</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/StackWithTextChildren-mount.html">5845</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/SwatchColorPicker-mount_base.html">12108</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/SwatchColorPicker-mount.html">12184</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/TagPicker-mount_base.html">3072</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/TagPicker-mount.html">3025</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/TeachingBubble-mount_base.html">96021</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/TeachingBubble-mount.html">96454</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/Text-mount_base.html">942</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/Text-mount.html">945</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/TextField-mount_base.html">1858</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/TextField-mount.html">1858</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/ThemeProvider-mount_base.html">1789</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/ThemeProvider-mount.html">1778</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/ThemeProvider-virtual-rerender_base.html">1250</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/ThemeProvider-virtual-rerender.html">1233</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">2531</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">2513</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/Toggle-mount_base.html">1255</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/Toggle-mount.html">1294</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/buttonNative-mount_base.html">634</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25957/perf-test/buttonNative-mount.html">644</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Pull request merged**